README.md

```diff
@@ -1 +1,129 @@
-# scalardb-samples
\ No newline at end of file
+# Scalar DB server Samples
+- This repository contains the sample application that will connect to `Scalar DB server` as backend. For using native `Scalar DB` library to connect to `Scalar DB` directly please refer to this document [`Getting Started`](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started.md).
+- `Scalar DB server` is a `gRPC` server that implements Scalar DB interface. More information about `Scalar DB server` can be found [`here`](https://github.com/scalar-labs/scalardb/tree/master/docs/scalardb-server.md)
+
+## Our sample application
+A simple electronic money cli app which has the following features:
+- Charge an `amount` to a `user_id`
+- Pay an `amount` from a `user_id` to another `user_id`
+
+## Prerequisites
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+## Setting up
+### Scalar DB server
+- In this sample we will use Cassandra as storage for our Scalar DB server. Cassandra will also run along with Scalar DB server in containers. The setting for our Scalar DB server is [`database.properties`](./database.properties)
+```
+# Comma separated contact points
+scalar.db.contact_points=cassandra
+
+# Port number for all the contact points. Default port number for each database is used if empty.
+scalar.db.contact_port=9042
+
+# Credential information to access the database
+scalar.db.username=cassandra
+scalar.db.password=cassandra
+
+# Storage implementation. Either cassandra or cosmos or dynamo or jdbc can be set. Default storage is cassandra.
+scalar.db.storage=cassandra
+```
+- To start our Scalar DB server, we use the below command (please note that we should wait around a bit more one minute to make sure every container is already up, because Scalar DB container has to wait for Cassandra container to be fully started)
```

Comment:
```suggestion
- To start our Scalar DB server, we use the below command (please note that we should wait around a bit more than one minute to make sure every container is already up, because Scalar DB container has to wait for Cassandra container to be fully started)
```